src/main/java/com/connexta/ingest/controllers/IngestController.java

```diff
@@ -35,7 +38,12 @@
 
   @Override
   public ResponseEntity<Void> ingest(
-      String acceptVersion, MultipartFile file, String correlationId, MultipartFile metacard) {
+      String acceptVersion,
+      @RequestHeader(name = "Last-Modified") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
```

Comment:
✏️ I think you can leave off the `@RequestHeader(name = "Last-Modified") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)` parts just like the rest of the method arguments because those should be included in the interface that this class extends.

Reply:
Good call, don't need the RequestHeader, but removing the datetime annotation causes the tests to fail. It seems spring doesn't bridge the gap between string and offsetdatetime object for us unless we tell it to expect a string with that format using this annotation and the specified pattern.